cran-comments.md

```diff
@@ -1,22 +1,53 @@
 ## Test environments
-* local OS X install, R 3.3.1
-* ubuntu 12.04 (on travis-ci), R 3.3.1
-* win-builder (devel and release)
+* local OS X install, R 3.4.1
+* ubuntu 12.04 (on travis-ci), R 3.4.1
+* win-builder (devel)
 
 ## R CMD check results
-0 errors | 0 warnings | 1 notes
-
-* License components with restrictions and base license permitting such:
-  MIT + file LICENSE
+0 errors | 0 warnings | 0 notes
 
 ## Reverse dependencies
 
-This was minor release that only added new features, but I ran revdep checks on the 160 reverse dependencies just to be sure (complete results at  https://github.com/hadley/scales/blob/master/revdep).  I saw no problems related to scales.
+I ran revdep checks on the 272 reverse dependencies (complete results at  https://github.com/hadley/scales/blob/master/revdep). There were issues with three packages; one of these issues appears unrelated to `scales`.
```

Comment:
In that case, I think you're probably providing a bit too much detail to CRAN. You can just say "the maintainers were informed of these issues on xxx xxxxx xxx". CRAN mainly wants to know that you've done due diligence.